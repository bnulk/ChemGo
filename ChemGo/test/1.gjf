%mem=1000MB
%chk=D:\ChemGo\ChemGo\test\LiuKun1.chk LiuKun2.chk
#p opt=ln sto-3g geom=connectivity <hf hessiann=1 hf> lambda=1.17
mecpfreq} scf=maxcycle=200 stepsize=1 task=mecp {cmd=g09 |

T1

0 1
 H                  0.00000000    0.00000000    0.00000000
 N                  0.00000000    0.00000000    1.25083700
 C                  1.29817076    0.00000000    1.66504761
 O                  1.84803831   -0.03092276    2.76999081

 1
 2 3 1.5
 3 4 2.0
 4

