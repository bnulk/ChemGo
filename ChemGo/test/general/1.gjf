%chk=D:\ChemGo\ChemGo\test\general\1.chk
# hf/3-21g

Title Card Required

0 1
 C              
 H                  1            B1
 H                  1            B2    2            A1
 C                  1            B3    3            A2    2            D1    0
 H                  4            B4    1            A3    3            D2    0
 H                  4            B5    1            A4    3            D3    0

   B1             1.07000000
   B2             1.07000000
   B3             1.35520000
   B4             1.07000000
   B5             1.07000000
   A1           119.88652694
   A2           119.88652694
   A3           120.22694612
   A4           119.88652694
   D1           180.00000000
   D2             1.6E-6
   D3           180.00000000


