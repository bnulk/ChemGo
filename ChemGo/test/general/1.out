 Entering Gaussian System, Link 0=g09
 Input=1.gjf
 Output=D:\ChemGo\ChemGo\test\general\1.out
 Initial command:
 C:\G09W\l1.exe .\gxx.inp D:\ChemGo\ChemGo\test\general\1.out /scrdir=.\
 Entering Link 1 = C:\G09W\l1.exe PID=     11672.
  
 Copyright (c) 1988,1990,1992,1993,1995,1998,2003,2009,2011,
            Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 09 program.  It is based on
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 09, Revision C.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, B. Mennucci, 
 G. A. Petersson, H. Nakatsuji, M. Caricato, X. Li, H. P. Hratchian, 
 A. F. Izmaylov, J. Bloino, G. Zheng, J. L. Sonnenberg, M. Hada, 
 M. Ehara, K. Toyota, R. Fukuda, J. Hasegawa, M. Ishida, T. Nakajima, 
 Y. Honda, O. Kitao, H. Nakai, T. Vreven, J. A. Montgomery, Jr., 
 J. E. Peralta, F. Ogliaro, M. Bearpark, J. J. Heyd, E. Brothers, 
 K. N. Kudin, V. N. Staroverov, T. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. Rendell, J. C. Burant, S. S. Iyengar, J. Tomasi, 
 M. Cossi, N. Rega, J. M. Millam, M. Klene, J. E. Knox, J. B. Cross, 
 V. Bakken, C. Adamo, J. Jaramillo, R. Gomperts, R. E. Stratmann, 
 O. Yazyev, A. J. Austin, R. Cammi, C. Pomelli, J. W. Ochterski, 
 R. L. Martin, K. Morokuma, V. G. Zakrzewski, G. A. Voth, 
 P. Salvador, J. J. Dannenberg, S. Dapprich, A. D. Daniels, 
 O. Farkas, J. B. Foresman, J. V. Ortiz, J. Cioslowski, 
 and D. J. Fox, Gaussian, Inc., Wallingford CT, 2010.
 
 ******************************************
 Gaussian 09:  IA32W-G09RevC.01 23-Sep-2011
                30-Oct-2019 
 ******************************************
 %chk=D:\ChemGo\ChemGo\test\general\1.chk
 ----------
 # hf/3-21g
 ----------
 1/38=1/1;
 2/12=2,17=6,18=5,40=1/2;
 3/5=5,11=9,16=1,25=1,30=1/1,2,3;
 4//1;
 5/5=2,38=5/2;
 6/7=2,8=2,9=2,10=2,28=1/1;
 99/5=1,9=1/99;
 -------------------
 Title Card Required
 -------------------
 Charge =  0 Multiplicity = 1
 Symbolic Z-Matrix:
 C
 H                    1    B1
 H                    1    B2       2    A1
 C                    1    B3       3    A2       2    D1       0
 H                    4    B4       1    A3       3    D2       0
 H                    4    B5       1    A4       3    D3       0
       Variables:
  B1                    1.07                     
  B2                    1.07                     
  B3                    1.3552                   
  B4                    1.07                     
  B5                    1.07                     
  A1                  119.88653                  
  A2                  119.88653                  
  A3                  120.22695                  
  A4                  119.88653                  
  D1                  180.                       
  D2                    0.                       
  D3                  180.                       
 
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0        0.000000    0.000000    0.000000
      2          1           0        0.000000    0.000000    1.070000
      3          1           0        0.927705    0.000000   -0.533164
      4          6           0       -1.170944    0.000000   -0.682243
      5          1           0       -1.170944    0.000000   -1.752243
      6          1           0       -2.098649    0.000000   -0.149080
 ---------------------------------------------------------------------
                    Distance matrix (angstroms):
                    1          2          3          4          5
     1  C    0.000000
     2  H    1.070000   0.000000
     3  H    1.070000   1.852234   0.000000
     4  C    1.355200   2.107479   2.103938   0.000000
     5  H    2.107479   3.055514   2.427032   1.070000   0.000000
     6  H    2.103938   2.427032   3.050630   1.070000   1.852234
                    6
     6  H    0.000000
 Stoichiometry    C2H4
 Framework group  C2H[SGH(C2H4)]
 Deg. of freedom     5
 Full point group                 C2H     NOp   4
 Largest Abelian subgroup         C2H     NOp   4
 Largest concise Abelian subgroup C2      NOp   2
                         Standard orientation:                         
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0        0.000000    0.677600    0.000000
      2          1           0       -0.924521    1.216266    0.000000
      3          1           0        0.927705    1.210764    0.000000
      4          6           0        0.000000   -0.677600    0.000000
      5          1           0        0.924521   -1.216266    0.000000
      6          1           0       -0.927705   -1.210764    0.000000
 ---------------------------------------------------------------------
 Rotational constants (GHZ):    146.1648694     29.8053186     24.7569804
 Standard basis: 3-21G (6D, 7F)
 There are    11 symmetry adapted basis functions of AG  symmetry.
 There are     2 symmetry adapted basis functions of BG  symmetry.
 There are     2 symmetry adapted basis functions of AU  symmetry.
 There are    11 symmetry adapted basis functions of BU  symmetry.
 Integral buffers will be    262144 words long.
 Raffenetti 1 integral format.
 Two-electron integral symmetry is turned on.
    26 basis functions,    42 primitive gaussians,    26 cartesian basis functions
     8 alpha electrons        8 beta electrons
       nuclear repulsion energy        33.3121054351 Hartrees.
 NAtoms=    6 NActive=    6 NUniq=    3 SFac= 4.00D+00 NAtFMM=   60 NAOKFM=F Big=F
 One-electron integrals computed using PRISM.
 NBasis=    26 RedAO= T  NBF=    11     2     2    11
 NBsUse=    26 1.00D-06 NBFU=    11     2     2    11
 Harris functional with IExCor=  205 diagonalized for initial guess.
 ExpMin= 1.83D-01 ExpMax= 1.72D+02 ExpMxC= 1.72D+02 IAcc=1 IRadAn=         1 AccDes= 0.00D+00
 HarFok:  IExCor=  205 AccDes= 0.00D+00 IRadAn=         1 IDoV= 1
 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 FoFCou: FMM=F IPFlag=           0 FMFlag=      100000 FMFlg1=           0
         NFxFlg=           0 DoJE=T BraDBF=F KetDBF=T FulRan=T
         Omega=  0.000000  0.000000  1.000000  0.000000  0.000000 ICntrl=     500 IOpCl=  0
         NMat0=    1 NMatS0=    1 NMatT0=    0 NMatD0=    1 NMtDS0=    0 NMtDT0=    0
         I1Cent=           4 NGrid=           0.
 Petite list used in FoFCou.
 Initial guess orbital symmetries:
       Occupied  (BU) (AG) (AG) (BU) (BU) (AG) (AG) (AU)
       Virtual   (BG) (AG) (BU) (BU) (AG) (BU) (AG) (AU) (BU) (BG)
                 (BU) (AG) (AG) (BU) (BU) (AG) (AG) (BU)
 The electronic state of the initial guess is 1-AG.
 Requested convergence on RMS density matrix=1.00D-08 within 128 cycles.
 Requested convergence on MAX density matrix=1.00D-06.
 Requested convergence on             energy=1.00D-06.
 No special actions if energy rises.
 Keep R1 ints in memory in canonical form, NReq=882313.
 SCF Done:  E(RHF) =  -77.5985302486     A.U. after    8 cycles
             Convg  =    0.6783D-08             -V/T =  2.0034

 **********************************************************************

            Population analysis using the SCF density.

 **********************************************************************

 Orbital symmetries:
       Occupied  (BU) (AG) (AG) (BU) (BU) (AG) (AG) (AU)
       Virtual   (BG) (AG) (BU) (BU) (AG) (BU) (AG) (BU) (AU) (BG)
                 (BU) (AG) (AG) (BU) (BU) (AG) (AG) (BU)
 The electronic state is 1-AG.
 Alpha  occ. eigenvalues --  -11.17045 -11.17036  -1.02773  -0.78970  -0.64985
 Alpha  occ. eigenvalues --   -0.57519  -0.51095  -0.36989
 Alpha virt. eigenvalues --    0.17733   0.29399   0.31929   0.34461   0.43495
 Alpha virt. eigenvalues --    0.52516   0.88490   0.93041   0.99707   1.07521
 Alpha virt. eigenvalues --    1.08837   1.12154   1.32652   1.36817   1.39393
 Alpha virt. eigenvalues --    1.66857   1.67162   1.95701
          Condensed to atoms (all electrons):
              1          2          3          4          5          6
     1  C    5.216938   0.393061   0.392911   0.525702  -0.048611  -0.049280
     2  H    0.393061   0.458806  -0.019290  -0.048611   0.002069  -0.001613
     3  H    0.392911  -0.019290   0.459999  -0.049280  -0.001613   0.002127
     4  C    0.525702  -0.048611  -0.049280   5.216938   0.393061   0.392911
     5  H   -0.048611   0.002069  -0.001613   0.393061   0.458806  -0.019290
     6  H   -0.049280  -0.001613   0.002127   0.392911  -0.019290   0.459999
 Mulliken atomic charges:
              1
     1  C   -0.430722
     2  H    0.215577
     3  H    0.215145
     4  C   -0.430722
     5  H    0.215577
     6  H    0.215145
 Sum of Mulliken atomic charges =   0.00000
 Mulliken charges with hydrogens summed into heavy atoms:
              1
     1  C    0.000000
     4  C    0.000000
 Sum of Mulliken charges with hydrogens summed into heavy atoms =   0.00000
 Electronic spatial extent (au):  <R**2>=             82.8165
 Charge=              0.0000 electrons
 Dipole moment (field-independent basis, Debye):
    X=              0.0000    Y=              0.0000    Z=              0.0000  Tot=              0.0000
 Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -12.1613   YY=            -12.2538   ZZ=            -15.7399
   XY=             -0.0133   XZ=              0.0000   YZ=              0.0000
 Traceless Quadrupole moment (field-independent basis, Debye-Ang):
   XX=              1.2237   YY=              1.1312   ZZ=             -2.3549
   XY=             -0.0133   XZ=              0.0000   YZ=              0.0000
 Octapole moment (field-independent basis, Debye-Ang**2):
  XXX=              0.0000  YYY=              0.0000  ZZZ=              0.0000  XYY=              0.0000
  XXY=              0.0000  XXZ=              0.0000  XZZ=              0.0000  YZZ=              0.0000
  YYZ=              0.0000  XYZ=              0.0000
 Hexadecapole moment (field-independent basis, Debye-Ang**3):
 XXXX=            -24.6472 YYYY=            -66.1550 ZZZZ=            -16.5497 XXXY=             -0.0028
 XXXZ=              0.0000 YYYX=             -0.0599 YYYZ=              0.0000 ZZZX=              0.0000
 ZZZY=              0.0000 XXYY=            -12.4127 XXZZ=             -7.5452 YYZZ=            -14.8553
 XXYZ=              0.0000 YYXZ=              0.0000 ZZXY=             -0.0014
 N-N= 3.331210543512D+01 E-N=-2.466247824861D+02  KE= 7.733174423682D+01
 Symmetry AG   KE= 3.931647584586D+01
 Symmetry BG   KE= 5.370401046130D-34
 Symmetry AU   KE= 1.920292677321D+00
 Symmetry BU   KE= 3.609497571363D+01
 1|1|UNPC-WIN-3U8THI5R4SL|SP|RHF|3-21G|C2H4|ADMINISTRATOR|30-Oct-2019|0
 ||# hf/3-21g||Title Card Required||0,1|C|H,1,1.07|H,1,1.07,2,119.88652
 694|C,1,1.3552,3,119.88652694,2,180.,0|H,4,1.07,1,120.22694612,3,0.000
 0016,0|H,4,1.07,1,119.88652694,3,180.,0||Version=IA32W-G09RevC.01|Stat
 e=1-AG|HF=-77.5985302|RMSD=6.783e-009|Dipole=0.,0.,0.|Quadrupole=0.849
 8304,-1.7508115,0.9009811,0.,-0.0250443,0.|PG=C02H [SGH(C2H4)]||@


 SUCCESS IS COUNTED SWEETEST
 BY THOSE WHO NE'ER SUCCEED.
 TO COMPREHEND NECTAR
 REQUIRES SOREST NEED.
           EMILY DICKINSON
 Job cpu time:  0 days  0 hours  0 minutes  3.0 seconds.
 File lengths (MBytes):  RWF=      5 Int=      0 D2E=      0 Chk=      1 Scr=      1
 Normal termination of Gaussian 09 at Wed Oct 30 18:56:09 2019.
